### PR TITLE
Add the initial implementation of the role

### PR DESCRIPTION
**Pull request by neumantm**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+This role installs and configures a [mumble](https://mumble.info/) server.
+
+## Requirements
+
+Debian.
+
+## Role Variables
+
+
+| Name| Required/Default | Description |
```
Comment (SlothOfAnarchy) [inline README.md]: Please indent the table so that it is readable without rendering

File: defaults/main.yml [hunk 2]
```diff
@@ -0,0 +1 @@
+---
```
Comment (SlothOfAnarchy) [inline defaults/main.yml]: This empty file can be deleted

**Review by SlothOfAnarchy — CHANGES_REQUESTED** [review]

**Review by SlothOfAnarchy — APPROVED** [review]

File: tasks/main.yml [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+    mumble_server_config: "{{ default_mumble_server_config | combine(mumble_server.config | default({}))}}"
+    mumble_server_ice_config: "{{ default_mumble_server_ice_config | combine(mumble_server.ice_config | default({}))}}"
+  template:
+    src: mumble-server.ini.j2
+    dest: /etc/mumble-server.ini
+  notify:
+    - Enable and restart mumble-server
+
+- name: Set superuser password
+  command:
```
Comment (dasJ) [inline tasks/main.yml:19]: Ummmmmmmmmmmmmm

```
command: murmurd -supw {{ … }}
become_user: murmurd
```

Comment (SlothOfAnarchy) [reply tasks/main.yml:19]: `no_log: True`

Comment (neumantm) [reply tasks/main.yml:19]: ehm right...

Comment (neumantm) [reply tasks/main.yml:19]: > `no_log: True`

This breaks the failed_when command

Comment (neumantm) [reply tasks/main.yml:19]: > > `no_log: True`
> 
> This breaks the failed_when command

Never mind, tried no_log under command instead of under name ...

**Pull request merged**